priv/schema/rabbitmq_management.schema

```diff
@@ -383,6 +383,12 @@ end}.
 {mapping, "management.uaa_location", "rabbitmq_management.uaa_location",
     [{datatype, string}]}.
 
+{mapping, "management.oauth2_scopes", "rabbitmq_management.oauth2_scopes",
```

Comment:
It's not clear to me what these scopes are used for. These mappings also lack tests, see the snippet file in `config_schema_SUITE_data`.

Reply:
I have updated PR description, there I explain purpose for this parameter.

Reply:
It was a hint that this should probably be renamed.

Reply:
@michaelklishin would you like to have this varaiable be named like _identityserver_scopes_? It's used only in this part of code referred to identity server implementation. I decided then to named it in generic manner because maybe in some day someone wish to use it in Singular implementation - there is already hard-coded _openid_ scope. Any suggestions will be kind see.

Reply:
`identityserver_scopes` sounds specific and follows the convention.

Right now the trend is towards having vendor-specific settings.

Reply:
Thank you, good to know about this practice. Done!